mysql-test/t/bug_ps7923.test

```diff
@@ -0,0 +1,36 @@
+--source include/count_sessions.inc
```

Comment:
`count_sessions.inc` probably not needed as we restart the server

Reply:
Removed.